README updates and some more comments

person-12 opened the pull request:
I don't think this needs a new version number, dunno what else to say. Just hopefully making it easier to install without contacting me.

person-12 committed:
Updated README and some more comments

basically what is says in the title, I don't think this needs a new version number

person-12 committed:
Update README.md

person-12 committed:
readme changes

FAQ section and rephrasing something else

SSkipr commented on README.md:
```diff
@@ -21,43 +21,59 @@ As previously stated, the application will recognize the following and send aler
  - **WEBHOOK_URL** This is the webhook url for sending screenshots. See [this tutorial](https://www.youtube.com/watch?v=xIZXDdVwNaE). This is optional.
 
+##### An example of configured variables
+ ![configured variables](https://cdn.discordapp.com/attachments/1349769873282175037/1431328076498862191/image.png?ex=68fd03a6&is=68fbb226&hm=9c0c338f9b292529fcdf2e836da63a79f31d2c9b3be92ba5352b5b7804f2571c)
```
Link DNE

SSkipr reviewed (changes requested):
Just that the Discord CDN expires... GitHub is a better fit as it doesn't expire (?)

person-12 commented:
oops misclick

person-12 committed:
Changed Image link

hopefully this works, the link says private-user-images but it seems to work when I'm not signed into github

person-12 commented:
I'll just grab the link from here then commit, cause this is public and should always be viewable
-# if you couldn't tell I have like 0 idea what I'm doing 
<img width="1420" height="101" alt="image" src="https://github.com/user-attachments/assets/fed2e97b-445e-4698-8079-57988402d987" />

person-12 committed:
Change image link again

maybe it'll work? idk what I'm doing really

SSkipr commented:
Alg lol. Might want to change your webhook url though...

SSkipr commented:
Thanks for your contribution, as always!

person-12 commented:
Most of the webhook (at least the last bit) is just random stuff, but thanks for pointing it out anyway
